spec/models/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+describe User do
+  describe '#is_passenger?' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.